Source/WTF/wtf/BumpPointerAllocator.h

```diff
@@ -98,16 +98,17 @@ class BumpPointerPool {
         return reinterpret_cast<char*>(reinterpret_cast<intptr_t>(allocation.base()) + allocation.size()) - size;
     }
 
-    BumpPointerPool(const PageAllocation& allocation)
+    BumpPointerPool(const PageAllocation& allocation, size_t remainingCapacity)
         : m_current(allocation.base())
         , m_start(allocation.base())
         , m_next(nullptr)
         , m_previous(nullptr)
         , m_allocation(allocation)
+        , m_remainingCapacity(remainingCapacity)
     {
     }
 
-    static BumpPointerPool* create(size_t minimumCapacity = 0)
+    static BumpPointerPool* create(size_t remainingCapacity, size_t minimumCapacity = 0)
```

Comment:
Isn't this really `maximumCapacity` within the context of this function? Probably also worth an `ASSERT(minimumCapacity <= maximumCapacity)`

Reply:
Keith commented elsewhere that this is due to a misunderstanding.